src/core/components/cards/cardCollapsible/cardCollapsible.test.tsx

```diff
@@ -0,0 +1,63 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import { CardCollapsible, type ICollapsibleCardProps } from './cardCollapsible'; // Ensure the correct path is used
+
+describe('<CardCollapsible />', () => {
```

Comment:
The tests seem to be more for the Collapsible as opposed to the Card Collapsible. I would go with something one ensuring that the collabsible is rendered in the card and perhaps one to check if the overlay is set properly, especially since you're having to access parent nodes and suppressing eslint issues.